.test-infra/jenkins/job_PerformanceTests_BigQueryIO_Python.groovy

```diff
@@ -90,7 +90,7 @@ PhraseTriggeringPostCommitBuilder.postCommitJob(
       executeJob(delegate, bqio_read_test)
     }
 
-CronJobBuilder.cronJob('beam_PerformanceTests_BiqQueryIO_Read_Python', 'H 15 * * *', this) {
+CronJobBuilder.cronJob('beam_PerformanceTests_BiqQueryIO_Read_Python', 'H H * * *', this) {
```

Comment:
what does H H mean?

Reply:
H 15 * * *' means the job will initiate every day 15:XX PM and currently all Python IO performance test is running on this schedule. As more tests being added we should distribute the time evenly throughout a day, which is 'H H * * *' --- both minute and hour are random